build/ci/jenkins_pipeline_build

```diff
@@ -3,10 +3,19 @@ pipeline {
         label 'docker-engine'
     }
     stages {
-        stage ('Run unit tests') {
-            steps {
-                sh 'mkdir -p build/_output/reports && chmod 777 build/_output/reports'
-                sh './build/ci/run_unittests.sh'
+        stage ('Running tests in parallel') {
```

Comment:
why is it in parallel?

Reply:
why not?

Reply:
The log of parallel parts can be mixed here?
E.g. can we see in logs/console something like this?
unitest log msg 1
olm validation log 1
unitest log msg 2
olm validation log 2

Reply:
depends on where you look.
I guess specifically console output will be mixed.
if you prefer to permanently double the build time for a serial log (even when there's no yaml change), I will remove it

Reply:
How much time it will add?

Reply:
currently less than 2 minutes

Reply:
2 min is not much. i prefer clear logs